### ISSUE TITLE
A11Y_Visual Studio Code Services_Docs User Guide_IntelliSense_ScreenReader:Column two is not read as header column and the data in column three doesn't announce it respective header value on using table keys. header.

### ISSUE THREAD

**Issue by v-jagansai** [microsoft/vscode-docs#5567]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yTCS;#BM_Visual Studio Code Services_Aug2022;#Visual Studio Code Services;#Win11;#FTP;#DesktopWeb;#A11ySev2;#WCAG4.2.3;#NVDA;#JAWS;#STP;#Chrome;

## Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Windows Version: Win11
Chrome Version: Version 103.0.5060.134 (Official Build) (64-bit)
NVDA Version:2021.3.5,JAWS

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Launch the application Url : https://code.visualstudio.com/
3. Navigate to "Docs" link.
4. From left navigation, select "User Guide".
5. Navigate to "IntelliSense" link.
6. Start Screen Reader .
7. Tab to table presented in the screen .
8. Observe that what Screen reader is announcing the content in column two. 

## Actual:
Column two is not read as header column and the data in column three doesn't announce it respective header value on using table keys. header.
Eg: On navigating to column two row two variable screen readers just announce row column and cell data it doesn't announce the word header same when it goes to cell row two column three .The Respective header reference is not read

Same issue is seen with Chrome+JAWS.

## Expected:
The data under  column two should be announced along with the word column header and the data in the column three should be read along with their respective column header references.

## User Impact:
Screen reader users will not get correct header references of the cell and will not know which is the header column.

## Attachment:
![Presentation6](https://user-images.githubusercontent.com/109065315/184312410-1fb181ae-1e59-46e4-84fd-485f0c53ddce.gif)
![file](https://user-images.githubusercontent.com/109065315/184351249-7a2a22f8-6374-490d-83d7-1f73ece82cf9.gif)

**Comment by gregvanl** [comment]
First row for column headers was missing. 
Fixed with https://github.com/microsoft/vscode-docs/commit/9360da4c7861288e791ab18275fbddb5da8f494d

**Comment by prabhudevu** [comment]
![5567_Issue fixed (1)](https://user-images.githubusercontent.com/98821792/216561544-2a32a5ad-4572-4727-96ca-6e5a7eacd172.png)
![5567_Issue fixed (2)](https://user-images.githubusercontent.com/98821792/216561605-e7c5ff60-375b-4af4-8954-064d220f4d2f.png)

I have verified the issue in the Env: https://code.visualstudio.com/docs/editor/intellisense Issue is fixed. closing the bug

**GithubTags**:#Closed;#Regressed:03-02-23;
